.circleci/config.yml

```diff
@@ -9,13 +9,15 @@ jobs:
       - run:
           name: Install dependencies from offline mirror
           command: yarn install --offline
+      - run:
+          name: Confirm project can build
+          command: |
+            yarn build:external
+            yarn build:internal
```

Comment:
If it runs both of these and then deploys, will it then deploy the internal build? Sorry not 100% familiar with how this works. 😄

Reply:
Good call! swapping